workflow/controller/operator_workflow_template_ref_test.go

```diff
@@ -496,7 +496,7 @@ const wfTmplWithVol = `
 apiVersion: argoproj.io/v1alpha1
 kind: WorkflowTemplate
 metadata:
-  name: workflow-template-whalesay-template
+  name: workflow-template-whalesay-template-with-volume
```

Comment:
I had to change the name to workaround this panic. It doesn't appear to change the behavior of the test though.

```
--- FAIL: TestWFTWithVol (0.00s)
panic: workflowtemplates.argoproj.io "workflow-template-whalesay-template" already exists [recovered]
	panic: workflowtemplates.argoproj.io "workflow-template-whalesay-template" already exists
```